sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyOperations.cs

```diff
@@ -19,4 +19,26 @@ public enum KeyOperations : uint
 
         All = uint.MaxValue
     }
+
+    public static class KeyOperationsExtensions
+    {
+        public static KeyOperations ParseFromString(string value)
+        {
+            switch (value)
+            {
+                case "encrypt": return KeyOperations.Encrypt;
+                case "decrypt": return KeyOperations.Decrypt;
+                case "sign": return KeyOperations.Sign;
+                case "verify": return KeyOperations.Verify;
+                case "wrapKey": return KeyOperations.Wrap;
+                case "unwrapKey": return KeyOperations.Unwrap;
+                default: return KeyOperations.Other;
+            }
+        }
+
+        public static string ParseToString(KeyOperations keyType)
+        {
+            return (keyType).ToString().ToLower();
```

Comment:
>return (keyType).ToString().ToLower(); [](start = 12, length = 38)

the values for this are not strictly lowercase, for instance unwrapKey.  I'm not sure if the service will enforce casing but we should probably be consistent with the docs.